src/components/App.tsx

```diff
@@ -70,7 +69,6 @@ const App: React.FC<AppProps> = (props) => {
 
   return (
     <div className={`WebsiteApp ${props.hasBanner ? 'withBanner' : ''}`}>
-      {!isElectron() && props.sound && <Audio track={`${process.env.PUBLIC_URL}/tone4.mp3`} play />}
```

Comment:
This is required to prevent the tab from halting the scripting if left unfocused

Reply:
This is no longer relevant because the 10hz is not getting detected so the throttling happens anyways :(

Reply:
Nevermind, this should be reverted. Check out https://github.com/eordano/background-throttle